tools/cloud-build/provision/main.tf

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+terraform {
+  backend "gcs" {
+    bucket = "hpc-toolkit-dev-automation"
```

Comment:
Could this be a parameter from cloud build or somewhere else? Ideally, we could avoid having project ids and GCS bucket names in this repo.

Reply:
Removed hardcoded bucket, `terraform init` will prompt user to enter one.